tools/test_containers_compose

```diff
@@ -9,4 +9,13 @@ for i in webui worker; do
     sudo docker-compose up -d &&
     (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
   ) || exit 1;
+
+  if [[ $i == "webui" ]]; then
+    (
+      docker-compose exec webui /usr/share/openqa/script/clone_job.pl --from https://openqa.opensuse.org --host http://nginx:9526 1738950 || echo "Error executing a job";
+      sleep 1;
+      docker-compose logs webui;
+      docker-compose logs webui | grep "Unable to wakeup scheduler: Connection refused" && exit 1
```

Comment:
```suggestion
      docker-compose exec webui openqa-clone-job --from https://openqa.opensuse.org --host http://nginx:9526 1738950 || echo "Error executing a job"
      sleep 1
      docker-compose logs webui
      docker-compose logs webui | grep "Unable to wakeup scheduler: Connection refused" && exit 1
```

Reply:
I put that in not ready because I am still working on that test. I'll move to WIP or draft